R/agglomerate.R

```diff
@@ -88,18 +94,22 @@
 #'   as \code{levels(f)}. (Default: \code{archetype = 1L}, which means the first
 #'   element encountered per factor level will be kept)
 #'
-#' @param mergeTree \code{TRUE} or \code{FALSE}: Should
-#'   \code{rowTree()} also be merged? (Default: \code{mergeTree = FALSE})
+#' @param agglomerate.tree \code{TRUE} or \code{FALSE}: Should
+#'   \code{rowTree()} also be merged? (Default: \code{agglomerate.tree = FALSE})
 #'
```

Comment:
Wondering if agglomerate.* parameters are too verbose. Should they be agg.* for instance agg.tree

Reply:
It is a bit verbose, on the other hand it is very clear. I might have a slight preference for the full name but no strong opinions. If shorter name is used then it could also be "aggl.*"

Reply:
Should we maintain or perhaps change from agglomerate.* to agg.*?

Reply:
We can keep them as agglomerate*

Reply:
ACK.